### DISCUSSION TITLE
Nginx in combination with DuckDNS - Who owns/manages the certificates?

### DISCUSSION THREAD

**Discussion by mgaeremynck** [hassio-addons/addon-nginx-proxy-manager#497] [Q&A]
Hello,

I installed the addon and can successfully connect to my HomeAssistant and all Proxy Hosts. I run this in combination with the DuckDNS addon to update my ip to my Hostname.

If I understand correctly, I need the DuckDNS addon to update my Hostname/IP. DuckDNS also generates a certificate (no wildcard certificate) and can't be disabled.

Nginx can manage the certificates with Wildcards but can't manage the IP Updates. 

Will this generate issues if both are "competing" for the same domain/certificate?

**Comment by frenck** [comment]
This add-on manages its own certificates. You don't need additional add-ons for that.

../Frenck

**Comment by mgaeremynck** [comment]
Hello,

Thanks for the feedback, this is a part of the answer. Great that Nginx manages the certificates but (at my knowledge) it doesn't manage the Dynamic IP updates. I still need DuckDNS to update my Dynamic IP and DuckDNS also want's to manage the certificate. 

Let's see what happens when the certificate expiries.

**Reply by TheS1ngularity** [reply]
> Hello,
> 
> Thanks for the feedback, this is a part of the answer. Great that Nginx manages the certificates but (at my knowledge) it doesn't manage the Dynamic IP updates. I still need DuckDNS to update my Dynamic IP and DuckDNS also want's to manage the certificate.
> 
> Let's see what happens when the certificate expiries.

Hi @mgaeremynck , in the end do did you face issues? And did you try without DuckDNS add-on, will the IP get updated if it changes without the DuckDNS add-on?

**Reply by mgaeremynck** [reply]
In the end I moved to the "Nginx Proxy Manager" addon (not the NGINX Home Assistant SSL proxy) as this provides more options.

For the DNS Updates I configured my router to manage this. Seems to work until now.
